lib/diagnostic.js

```diff
@@ -8,22 +8,47 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
+  let minNum = arguments[0];
 
+  for (let i = 0; arguments[i].length; i++) {
```

Comment:
Since ```arguments``` is expected to be empty or an array containing at least one number, ```arguments[i].length``` is attempting to find a ```length``` property on a number and it will evaluate to ```undefined```.  Thus, this loop will never run.